lib/services/credentials/credential.service.js

```diff
@@ -95,7 +94,7 @@ s.insertCredential = function (id, type, credentialDetails) {
 
       return Promise.all([validateNewCredentialScopes(credentialConfig, credentialDetails),
         validateAndHashPassword(credentialConfig, credentialDetails),
-        validateNewCredentialProperties(credentialConfig, _.omit(credentialDetails, ['scopes']))
+        validateNewCredentialProperties(credentialConfig, credentialDetails)
```

Comment:
I guess it's just ok to pass this additional data to this function

Reply:
yes, I added scopes filtering to the function itself, it was 5-6 places with _.omit scopes.